addons/payment_stripe/views/payment_provider_views.xml

```diff
@@ -47,7 +47,7 @@
                             type="object"
                             name="action_stripe_verify_apple_pay_domain"
                             class="btn btn-primary"
-                            invisible="not allow_express_checkout"/>
+                            invisible="not allow_express_checkout or code != 'stripe'"/>
```

Comment:
If it's possible, I'd rather put a `t-if` on the field at line 42. There is no need to replace the field on all providers when Stripe is installed.
```suggestion
                            invisible="not allow_express_checkout/>
```
```xml
<field t-if="code='stripe'" name="allow_express_checkout" position="replace">
```